lambda_functions/ml_service/GetKeyPhrases.py

```diff
@@ -11,6 +14,31 @@
 comprehend = boto3.client(service_name='comprehend', region_name='eu-central-1')
 
 
+def post_TopicalPageRank(language, data):
+    stage = os.environ['STAGE']    
+    if language == "de":
+        url="https://8wfgkfs2wc.execute-api.eu-central-1.amazonaws.com/"+stage+"/models/TopicalPageRank"
```

Comment:
Please use a custom domain for your api gateway and specify the stage as a subdomain (e.g. api.<stage>.detektivkollektiv.org/ml_model_service/models/TopicalPageRank"

We can set this up together, if you want to :)

Reply:
That would be great, as I have no idea how I need to configure that